src/nuxt.config.js

```diff
@@ -249,6 +251,7 @@ const nuxtConfig = {
           'https://likecoin.github.io',
           'verify.walletconnect.com',
           'www.facebook.com',
+          '*',
```

Comment:
Prefer just allowing youtube domain, * basically mean allow all which the the iframe-src list useless

Reply:
[updated](https://github.com/likecoin/liker-land/pull/1464/commits/3e914f84134f07b720f26cb0272156b045c34b4b)